inc/Nav_Menus/Component.php

```diff
@@ -26,7 +27,12 @@
  */
 class Component implements Component_Interface, Templating_Component_Interface {
 
-	const PRIMARY_NAV_MENU_SLUG = 'primary';
```

Comment:
This must not be removed and should be reused, also since we have to work backward-compatible from now on.